### PR TITLE
Remove sanity checking

### PR DESCRIPTION
**Pull request by ao508**
⚠️ **TODO: Need to decide on whether we still want to have logging done in cmo metadb to record duplicate requests that come through** 
----

https://github.com/mskcc/cmo-metadb/pull/316/files#diff-2fee701fd9cface3562b4c12c46a0c818112e6f5d522fd30719778d703b54369R69

Alternative solution might be to publish to a "logging" topic that will be handled by the request filter/sample validator which will then log the request.

### CODE REVIEW

**Comment by ao508** [comment]
@divyamadala30 the review can wait until the request-reply messaging is implemented in the messaging lib

File: service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java [hunk 1]
```diff
@@ -178,12 +158,6 @@ public void onMessage(Message msg, Object message) {
                     messageHandlingService.newRequestHandler(metaDbRequest);
                 } catch (Exception e) {
                     LOG.error("Exception during processing of request on topic: " + IGO_NEW_REQUEST_TOPIC, e);
-                    try {
-                        requestStatusLogger.logRequestStatus(message.toString(),
-                                RequestStatusLogger.StatusType.REQUEST_PARSING_ERROR);
```
Comment (divyamadala30) [inline service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java:183]: Shouldn't we log parsing errors

**Review by divyamadala30 — COMMENTED** [review]
Question about using timestamps to find duplicates in RequestServiceImpl: Are we still using that even after setting up ids to catch duplicates?

File: service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java (hunk 1 above)
Comment (ao508) [reply service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java:183]: those would be handled in the request filter app - I have a separate PR w/some changes to that

Comment (ao508) [reply service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java:183]: @divyamadala30 here it is https://github.com/mskcc/cmo-metadb-request-filter/pull/6/files

Comment (divyamadala30) [reply service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java:183]: Doesn't this catch errors while converting the message into a java object?

**Review by divyamadala30 — APPROVED** [review]

**Pull request merged**